backend/migrations/versions/bd271a2780b2_init_database.py

```diff
@@ -1275,7 +1263,6 @@ def upgrade():
 
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.drop_table('worksheet_share')
```

Comment:
If we are deleting everything related to worksheet shares, should we delete the table as well? 
I mean put this line `op.drop_table('worksheet_share')` into modularization migration script

Reply:
I moved it